docs/upgrade/11_to_12/workflows_mysql5.py

```diff
@@ -0,0 +1,425 @@
+# Upgrade script for MySQL databases
+# Requires Python3 to run
+# Required packages:
+#   $ pip install mysql-connector-python
+# Set vars to point to your database
+# Run on commandline: "python3 workflow_db_upgrade.py"
+# WARNING: THIS SCRIPT DELETES DATA. CREATE A BACKUP BEFORE RUNNING
+
+# Module Imports
+import mysql.connector
+from mysql.connector import Error
+import sys
+import xml.etree.ElementTree as ET
+import distutils.util
+from distutils.util import strtobool
+from datetime import datetime
+
+# Vars
+user = "opencast"
+password = "dbpassword"
+host = "127.0.0.1"
+database = "opencast"
+
+# Constants
+workflow_table_name = "oc_workflow"
+workflow_configuration_table_name = "oc_workflow_configuration"
+workflow_operation_table_name = "oc_workflow_operation"
+workflow_operation_configuration_table_name = "oc_workflow_operation_configuration"
+
+# DB functions
+def create_connection(host_name, user_name, user_password, db_name):
+    connection = None
+
+    try:
+        connection = mysql.connector.connect(
+            host=host_name,
+            user=user_name,
+            passwd=user_password,
+            database=db_name
+        )
+        connection.row_factory = lambda cursor, row: row[0]
+        print("Connection to MySQL DB successful")
+    except Error as e:
+        print(f"The error '{e}' occurred")
+    return connection
```

Comment:
Why catch this and return `None`? Shouldn't the script immediately fail if a connection could not be established? I suggest removing the `try … except …` More so since you just catch the error to then print the error anyway. Something which would happen automatically.